fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalOlapTableSink.java

```diff
@@ -202,9 +202,12 @@ public PhysicalProperties getRequirePhysicalProperties() {
             DistributionInfo distributionInfo = targetTable.getDefaultDistributionInfo();
             if (distributionInfo instanceof HashDistributionInfo) {
                 // Do not enable shuffle for duplicate key tables when its tablet num is less than threshold.
-                if (targetTable.getKeysType() == KeysType.DUP_KEYS
-                        && distributionInfo.getBucketNum() < Config.min_tablets_for_dup_table_shuffle) {
-                    return PhysicalProperties.ANY;
+                if (targetTable.getKeysType() == KeysType.DUP_KEYS) {
+                    final long partitionNums = targetTable.getPartitionInfo().getAllPartitions().size();
```

Comment:
```suggestion
                    final long partitionNums = partitionIds.size();
```
`partitionIds` are the current partitions to be imported？If that's the case，using the size of `partitionIds` would be better.